clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/BlockDeviceConfig.groovy

```diff
@@ -99,6 +99,11 @@ class BlockDeviceConfig {
       "i3.16xlarge" : enumeratedBlockDevicesWithVirtualName(8),
       "i3.metal"    : enumeratedBlockDevicesWithVirtualName(8),
 
+      "i3en.large"  : enumeratedBlockDevicesWithVirtualName(1),
+      "i3en.xlarge" : enumeratedBlockDevicesWithVirtualName(1),
+      "i3en.2xlarge": enumeratedBlockDevicesWithVirtualName(1),
```

Comment:
```suggestion
      "i3en.2xlarge": enumeratedBlockDevicesWithVirtualName(2),
```

Reply:
I think, based on this table: https://aws.amazon.com/ec2/instance-types/i3en/

Reply:
worth adding the rest of the i3en types from that page while you are in here?

Reply:
👍